far_mar.rb

```diff
@@ -0,0 +1,12 @@
+# gems your project needs
+require 'csv'
+
+module FarMar
+
+
+end
+
+require_relative './lib/markets'
```

Comment:
Let's rename these files so they match the class name. The convention is to use singular class and file names (i.e., `Market` and `market.rb`).